analytics_data_api/v0/documents.py

```diff
@@ -160,6 +160,8 @@ def get_users_in_course(
             }
             for sort_policy in sort_policies
         ])
+        total = search_request.count()
```

Comment:
Can you add some comments to these two lines? Does counting the search_requests generate unnecessary calls to ES?
Why do we need to do the next line? It's very unintuitive that the `search_request` first only include the first 10, not by default all of them.